src/spidevioctl.rs

```diff
@@ -108,6 +108,19 @@ impl<'a, 'b> spi_ioc_transfer<'a, 'b> {
         }
     }
 
+    /// Create a read/write transfer using the same buffer for reading
+    /// and writing (in-place transfer).
+    pub fn read_write_in_place(buf: &'a mut [u8]) -> Self {
```

Comment:
What about renaming the function to `write_read_in_place`? Viewed from the "outside", `buf` contains the data to write before the call and in case of success the data read via SPI after the call.

```suggestion
    /// Create a write/read transfer using the same buffer for writing
    /// and reading (in-place transfer).
    pub fn write_read_in_place(buf: &'a mut [u8]) -> Self {
```

I also see that there is already `read_write(tx_buf, rx_buf)` with the buffers in opposite order oft the verbs in the function name. What about adding `write_read` and deprecating `read_write` too?

Reply:
I looked around a bit how other libraries call these functions, and didn't find any prevalent pattern. If anything, variations of transfer/xfer show up most often, but that term is already used for both `SpidevTransfer` and the `transfer` function that is used to execute a `SpidevTransfer`, so using that here as well would be too confusing.

With SPI the write and the read operation (on the bus) happen at the same time. Therefore I don't think calling it `write_read` instead of `read_write` provides a strong enough advantage to justify the deprecation/renaming dance.